index.html

```diff
@@ -28,9 +28,10 @@ <h1 class="title">
                 </h1>
                 <h2 class="subtitle">
                     web apps shared in a chat
-                    <br>
-                    no logins, coins or platforms 🎉
                 </h2>
+                <p>
+                    🎉 no <strike>logins</strike>, <strike>coins</strike> or <strike>platforms</strike>
```

Comment:
hm, i read that as "no or" now :)

maybe write "🎉 no logins, no coins, no platforms"

no, no, no :)